### ISSUE TITLE
Change dropdown on focus text colour to black

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1107]
As a result of recent high contrast changes, it has been noticed that the colour contrast when a disggregation control has focus no longer meets 'AA' compliance.

Recommendation: Change text to black on focus for 
![Screenshot (150)_LI](https://user-images.githubusercontent.com/60919820/108985450-f8660f00-7688-11eb-9724-a50d11c6a3af.jpg)
high contrast mode.
